libpod/oci.go

```diff
@@ -268,6 +270,9 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string) (er
 	if ctr.config.ConmonPidFile != "" {
 		args = append(args, "--conmon-pidfile", ctr.config.ConmonPidFile)
 	}
+	if r.syslog {
```

Comment:
We should have `log-level=debug` imply this

Reply:
I am fine with that if it makes more sense.  Although I am think we should ouput something to the user like this just before launching conmon. 
```
logrus.Debug("conmon outputs additional information to syslog when in Debug mode") 
```
So the user has an idea to look there.

Reply:
> Although I am think we should ouput something to the user like this...

This sounds better to me.  I'd rather not mix log level with the log channel.